File: CSharp/1196.cs
using System;
using System.Linq;
class Program
{
    static void Main(string[] args)
    {
        char[] letras="`1234567890-=QWERTYUIOP[]\\ASDFGHJKL;'ZXCVBNM,./".ToCharArray();
        int indice=0;
        string x;
        while(true)
        {
            x=Console.ReadLine();
            if(string.IsNullOrEmpty(x))
            {break;}
        char[] frase=x.ToCharArray();

        for(int i=0;i<frase.Length;i++)
        {
            indice = Array.IndexOf(letras,frase[i]);
            
            if(indice!=-1)
            {
            frase[i]=letras[indice-1];
            }
        }
        Console.WriteLine(string.Join("",frase));
    }}
}

File: Python/1196.py
letras = "`1234567890-=QWERTYUIOP[]\\ASDFGHJKL;'ZXCVBNM,./"

while True:
    try:
        x = input()
        if not x:
            break

        frase = list(x)

        for i in range(len(frase)):
            indice = letras.find(frase[i])
            
            if indice != -1:
                frase[i] = letras[indice - 1]

        print(''.join(frase))
    
    except EOFError:
        break
